app/constants.py

```diff
@@ -5,6 +5,10 @@
 from phonenumbers import PhoneNumberFormat
 from pydantic_extra_types.phone_numbers import PhoneNumber
 
+# this should be 2886 to retry for a full 24 hours
+# setting to 3 for now so local testing doesn't take too long
+MAX_RETRIES = 3
```

Comment:
This seems like a good reason to read this value from the environment.

Reply:
Good call.